R/marginal_model.R

```diff
@@ -1,41 +1,61 @@
 #' Prepare marginal model to pass through to `brms`
 #'
 #' @param data A `data.frame` containing line list data
+#' @param ... Additional arguments passed to methods.
 #' @family marginal_model
 #' @export
-as_epidist_marginal_model <- function(data) {
+as_epidist_marginal_model <- function(data, ...) {
   UseMethod("as_epidist_marginal_model")
 }
 
 #' The marginal model method for `epidist_linelist_data` objects
 #'
 #' @param data An `epidist_linelist_data` object
+#' @param obs_time_threshold Ratio used to determine threshold for setting
```

Comment:
Will want to document how this approximation changes accuracy

Reply:
it doesn't in this conservative case